docs/using-scylla/features-ent.rst

```diff
@@ -33,13 +33,12 @@ Scylla Enterprise Features
   :id: "getting-started"
   :class: my-panel
 
-  The following are **new features** for Scylla Scylla Enterprise 2022.1.x:
+  The following are **new features** in ScyllaDB Enterprise 2022.1.x:
```

Comment:
ScyllaDB is the name of the company, the product is just Scylla, no?

Reply:
@denesb This has changed and the new policy is to use ScyllaDB referring to the product.

Reply:
Ok, good to know.